lib/tasks/exchange_rates.rake

```diff
@@ -0,0 +1,104 @@
+namespace :exchange_rates do
+  desc 'Remove and re-build old monthly rates inclusive of the provided data (does not download from XE)'
+  task rebuild_old_monthly_rates: :environment do
+    raise ArgumentError, 'Supply an MONTH_START_PERIOD env var' if ENV['MONTH_START_PERIOD'].blank?
+    raise ArgumentError, 'Supply an YEAR_START_PERIOD env var' if ENV['YEAR_START_PERIOD'].blank?
+    raise ArgumentError, 'Supply an MONTH_END_PERIOD env var' if ENV['MONTH_END_PERIOD'].blank?
+    raise ArgumentError, 'Supply an YEAR_END_PERIOD env var' if ENV['YEAR_END_PERIOD'].blank?
+    raise ArgumentError, 'Supply an CURRENCY_CODE env var' if ENV['CURRENCY_CODE'].blank?
+
+    from_date = Date.new(ENV['YEAR_START_PERIOD'].to_i, ENV['MONTH_START_PERIOD'].to_i).beginning_of_month
+    to_date = Date.new(ENV['YEAR_END_PERIOD'].to_i, ENV['MONTH_END_PERIOD'].to_i).end_of_month
+
+    if ENV['CURRENCY_CODE']
```

Comment:
This is used if there is an erroneous currency code in the DB. This will remove it and re-build the monthly files.

This rake task might be used without removing rates if a new way of force downloading the live rates from XE or there is something wrong with the files we have.